src/IAM/Application/Logout/RefreshTokenStoreInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Twitter\IAM\Application\Logout;
+
+interface RefreshTokenStoreInterface
```

Comment:
Is it a repository? If so, please rename it respectfully. For example:

```suggestion
interface RefreshTokenRepository
```

No need for the `Interface` prefix.